src/SearchWrapper.cpp

```diff
@@ -224,8 +322,10 @@ parseQuery(Rcpp::List & query)
 	    aOP = Xapian::Query::OP_ELITE_SET;
 	else if (qOP == "OP_FILTER")
 	    aOP = Xapian::Query::OP_FILTER;
+#if defined(XAPIAN_RANGE_PROCESSOR_SUPPORT)
```

Comment:
One final thing before merging: I think this should have a separate define, `XAPIAN_OP_MAX_SUPPORT` or similar. It feels odd to guard this based on a name that has nothing to do with it. `OP_MAX` was apparently introduced in 1.3.1.